docs/the-new-architecture/landing-page.md

```diff
@@ -3,11 +3,26 @@ id: landing-page
 title: Introduction
 ---
 
-This section is the entry point for the new guide’s documentation.
-It contains some basic information about the New Architecture: its pillars, the version from which it has been made available and other very high-level information. Then, it should present the guide’s structure itself.
+Starting in version 0.68, React Native provides a New Architecture that improves several aspects of how JavaScript code and host platforms can integrate, such as:
 
-This section should also contains a sort of Driving guide based on the user use-cases:
+- synchronously measure and render React Native views at the same time as native views
+- support for prioritization of events that update the app's layout, like user interactions
+- improved type safety and consistency of communication between JavaScript and native code
 
-- New user: link to how to use the new [app template](use-app-template)
-- New Library developer: link to the [pillars](pillars)
-- Old library developer/app developer: link to [the migration guide](../new-architecture-intro)
+This is made possible by these [Pillars of the New Architecture](pillars):
+
+- [Fabric renderer and components](pillars-fabric-components), and the capabilities they offer for layout and interaction events
+- [TurboModules](pillars-turbo-modules), which supports faster loading of and communication with native code
+- [CodeGen](pillars-codegen), which generates code you can use to interface with native modules, via static typing in JavaScript
+
+## Get started with the New Architecture
+
+### For app developers
+
+To **create a new app** using the New Architecture, head over to [Creating a New Architecture App](use-app-template.md), which will get you up and running in a few easy steps with the new app template.
+
+To **migrate an existing app** to the New Architecture, follow [Adopting the New Architecture](../new-architecture-intro.md).
+
+### For library maintainers
+
+If you are adding support for the New Architecture to a library, head to the [Pillars]([pillars]) section to learn more about supporting apps that have migrated to the New Architecture, while still maintaining backwards compatibility for apps that haven't.
```

Comment:
Backward compatibility will have a separate section.
I think that here we are missing two paths:
* Do you already have a library and you want to migrate to the new architecture => go to the library migration guide
* Do you already have a library and it has to work on both the architectures => go to the backward compatibility guide.

We kept the backward compatibility separated with respect to the creation of a TM or FC because it requires some extra steps. Going forward, we would like to have all the users on the New Architecture and to have only TM and FC that works with the New Architecture.